wasp/wasp.py

```diff
@@ -504,4 +507,27 @@ def schedule(self, enable=True):
 
         self._scheduling = enable
 
+
+    def set_theme(self, new_theme: bytes):
+        self._theme = array.array('H', new_theme)
+    def theme(self, theme_part: str) -> int:
+        """Returns the relevant part of theme. For more see ../tools/themer.py"""
+        theme_parts = ("ble",
+                       "scroll-indicator-up",
+                       "scroll-indicator-down",
```

Comment:
Why would up/down ever need to be be different colours?

Reply:
idk, in the next revision they won't be.